apps/nrm/views/content/token-invalid.md

```diff
@@ -1 +1 @@
-To start or return to a report, you’ll need to request secure access again via the report modern slavery start page on [GOV.UK](/start).
+To start or return to a report, you’ll need to request secure access again via the report modern slavery start page on <a class="govuk-link" href="/start">GOV.UK.</a>
```

Comment:
Would it be more readable to use md syntax in this file and just extend any necessary govuk classes via the css?

Reply:
Replaced the <a> tag with md syntax.